allegro.cpp

```diff
@@ -3466,7 +3466,7 @@ void Alg_seq::beat_to_measure(double beat, int *measure, double *m_beat,
     }
     bpm = prev.num * 4 / prev.den;
     m = m + (beat - prev.beat) / bpm;
-    *measure = m;
+    *measure = static_cast<int>(m);
```

Comment:
same here: is there a problem with (int) m?

Reply:
I reached for C++ casts first since this is a `.cpp` file, but I can do that if you'd like.